File: src/main/java/com/mintos/weather/dto/WeatherDto.java
package com.mintos.weather.dto;

import java.math.BigDecimal;
import java.util.UUID;

public record WeatherDto(
    UUID id,
    String condition,
    BigDecimal temperatureCelsius,
    BigDecimal windMpH,
    String windDirection,
    Long lastUpdatedEpoch
) {
}

File: src/main/kotlin/com/mintos/weather/dto/WeatherDto.kt
package com.mintos.weather.dto

import java.math.BigDecimal
import java.util.UUID

data class WeatherDto(
    val id: UUID,
    val condition: String,
    val temperatureCelsius: BigDecimal,
    val windMpH: BigDecimal,
    val windDirection: String,
    val lastUpdatedEpoch: Long,
)
